tests/unit/test_kubernetes_control_plane.py

```diff
@@ -525,6 +569,74 @@ def test_psp_config_1_25():
     hookenv.log.assert_called_with("Creating pod security policy resources.")
 
 
+# get_kube_system_pods_not_running and get_dns_provider are mocked since they contain hookenv.config calls
+@mock.patch(
+    "reactive.kubernetes_control_plane.get_kube_system_pods_not_running",
+    return_value=[],
+)
+@mock.patch("reactive.kubernetes_control_plane.get_dns_provider")
+@mock.patch(
+    "reactive.kubernetes_control_plane.hookenv.config",
+    side_effect=[False, True, "10.152.183.0/24", False],
+)
+def test_ignore_missing_cni(mock_config, mock_dns_provider, mock_get_pods):
```

Comment:
this was in the wrong spot before, moved it up with the rest of the tests, and mocked out the dns call and the kube_system pod check since they contain hookenv.config calls and its easier if those are mocked out of the way